cli/repl.js

```diff
@@ -0,0 +1,27 @@
+var repl = require('repl')
+
+var AccountAdmin = require('@hoodie/account-client/admin')
```

Comment:
I thought a bit more about this, and I think we should not use the `@hoodie/account-client/admin` after all, because it requires a server to be running somewhere. We could start the server in the background and then open the REPL (and I think this will probably be the better thing to do later), but for now we could also use https://github.com/hoodiehq/hoodie-account-server-api. Pass in `'_users'` as for the `usersDb` option and `'secret'` for the `secret` option for now. For the `PouchDB` option, do what we currently do in https://github.com/hoodiehq/hoodie/blob/be9e92fa700cc028d3c4c469f595963cfa4f1bca/server/index.js#L23-L61